starboard/shared/posix/socket_send_to.cc

```diff
@@ -72,6 +73,10 @@ int SbSocketSendTo(SbSocket socket,
 
     ssize_t bytes_written = sendto(socket->socket_fd, data, data_size,
                                    kSendFlags, sockaddr, sockaddr_length);
+#else
+    ssize_t bytes_written =
+        sendto(socket->socket_fd, data, data_size, kSendFlags, nullptr, 0);
```

Comment:
This could technically fail if Cobalt ever sends on a socket that isn't connected, but it's probably fair to say that it doesn't. 

I think DNS lookups could be done like that, but we don't resolve ourselves and let the OS DNS resolver do that communication.

In the future, that may change, but that will be on Starboard 17 where the issue that this QUIRK addresses no longer exists.